src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/util/GraphQLRequestParamUtils.java

```diff
@@ -79,28 +81,27 @@ public static boolean isGraphQLContentType(final String contentType) {
      * @throws RuntimeException if JSON serialization fails for some reason due to any runtime environment issues
      */
     public static String toPostBodyString(final GraphQLRequestParams params) {
-        final ObjectMapper mapper = new ObjectMapper();
-        final ObjectNode postBodyJson = mapper.createObjectNode();
-        postBodyJson.set(OPERATION_NAME_FIELD,
-                JsonNodeFactory.instance.textNode(StringUtils.trimToNull(params.getOperationName())));
-
-        if (StringUtils.isNotBlank(params.getVariables())) {
-            try {
-                final ObjectNode variablesJson = mapper.readValue(params.getVariables(), ObjectNode.class);
-                postBodyJson.set(VARIABLES_FIELD, variablesJson);
-            } catch (JsonProcessingException e) {
-                log.error("Ignoring the GraphQL query variables content due to the syntax error: {}",
-                        e.getLocalizedMessage());
+        final JsonFactory jsonFactory = new JsonFactory();
```

Comment:
https://fasterxml.github.io/jackson-core/javadoc/2.6/com/fasterxml/jackson/core/JsonFactory.html

Please move the factory to a static field

Reply:
Done in 9e63931ee11f206fa846bd8277bfe8b8bd6cb13d. Thanks!